lib/elastic_record/index/settings.rb

```diff
@@ -18,8 +18,9 @@ def settings
           end
       end
 
-      def update_settings(index_name = alias_name)
-        connection.json_put "/#{index_name}/_settings", settings
+      def update_settings(index_name = alias_name, settings: nil)
+        self.settings = settings if settings
```

Comment:
Can this be defaulted in the method signature? `settings: default_settings`

Reply:
No, the point is really to call `settings=` with the supplied values so the local instance variable is up-to-date. `settings=` has logic for merging into the entire set of settings for the index.